example-application/test/basic-tests.test.js

```diff
@@ -35,9 +31,7 @@ beforeEach(() => {
     name: "John",
   });
 
-  if (sinonSandbox) {
-    sinonSandbox.restore();
-  }
+  sinon.restore();
```

Comment:
IMO this should be in `afterEach`
1. More reasonable.
2. Sometimes you may want to declare some stubs in the `beforeEach` and the restore will remove them